[2020-07-31 05:54:33,451] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: DAG_jira.get_issues 2020-07-30T21:43:00+00:00 [queued]>
[2020-07-31 05:54:33,484] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: DAG_jira.get_issues 2020-07-30T21:43:00+00:00 [queued]>
[2020-07-31 05:54:33,485] {taskinstance.py:866} INFO - 
--------------------------------------------------------------------------------
[2020-07-31 05:54:33,486] {taskinstance.py:867} INFO - Starting attempt 1 of 1
[2020-07-31 05:54:33,486] {taskinstance.py:868} INFO - 
--------------------------------------------------------------------------------
[2020-07-31 05:54:33,514] {taskinstance.py:887} INFO - Executing <Task(PythonOperator): get_issues> on 2020-07-30T21:43:00+00:00
[2020-07-31 05:54:33,521] {standard_task_runner.py:53} INFO - Started process 8440 to run task
[2020-07-31 05:54:33,656] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: DAG_jira.get_issues 2020-07-30T21:43:00+00:00 [running]> 02f6ef7ded4c
[2020-07-31 05:54:33,754] {logging_mixin.py:112} INFO - [2020-07-31 05:54:33,753] {base_hook.py:84} INFO - Using connection to: id: jira_issues. Host: https://jira.huhta-lauri.dev, Port: None, Schema: , Login: LauriHu, Password: XXXXXXXX, extra: None
[2020-07-31 05:54:33,760] {logging_mixin.py:112} INFO - [2020-07-31 05:54:33,759] {http_hook.py:136} INFO - Sending 'GET' to url: https://jira.huhta-lauri.dev/rest/api/2/search?jql=updated>"2020-07-17 21:54"
[2020-07-31 05:54:33,796] {logging_mixin.py:112} WARNING - /usr/local/lib/python3.7/site-packages/urllib3/connectionpool.py:1004: InsecureRequestWarning: Unverified HTTPS request is being made to host 'jira.huhta-lauri.dev'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning,
[2020-07-31 05:54:34,064] {logging_mixin.py:112} INFO - [2020-07-31 05:54:34,064] {jira_datapipeline.py:54} INFO - Saving dag_data/dag_jira_data.json
[2020-07-31 05:54:34,073] {python_operator.py:114} INFO - Done. Returned value was: None
[2020-07-31 05:54:34,091] {taskinstance.py:1048} INFO - Marking task as SUCCESS.dag_id=DAG_jira, task_id=get_issues, execution_date=20200730T214300, start_date=20200731T055433, end_date=20200731T055434
[2020-07-31 05:54:43,422] {logging_mixin.py:112} INFO - [2020-07-31 05:54:43,421] {local_task_job.py:103} INFO - Task exited with return code 0
